plugins/syslogng.yaml

```
@@ -0,0 +1,71 @@
+    default: "UTC"
+    relevant_if:
+      protocol:
+        equals: "rfc3164 (BSD)"
+
+# Set Defaults
+# {{$listen_address := default "0.0.0.0:514" .listen_address}}
+# {{$connection_type := default "udp" .connection_type}}
+# {{$protocol := default "rfc5424 (IETF)" .protocol}}
+# {{$location := default "UTC" .location}}
+
+# Pipeline Template
+pipeline:
+# {{ if eq $connection_type "udp" }}
+  - id: syslogng_input
```

Id's should be unique.
```suggestion
  - id: syslogng_udp_input
```